lib/src/Uncertainty/Distribution/RandomMixture.cxx

```diff
@@ -2241,7 +2240,6 @@ Scalar RandomMixture::computeProbability(const Interval & interval) const
   // Special case for combination containing only one contributor
   if (isAnalytical_)
   {
-    cdfEpsilon_ = ResourceMap::GetAsScalar( "RandomMixture-DefaultCDFEpsilon" );
```

Comment:
This key equals 1e-10 whereas default CDF epsilon equals 1e-14, you may move this initialization in constructor.

Reply:
It was already initialized in the ctor, I guess it was to reinitialize the value after altering it.

Reply:
you're right

Reply:
now I see RandomMixture also has its own pdfPrecision, cdfPrecision as mutable and use the same pattern.

Reply:
IMHO you can fix [pc]dfEpsilon_ in this PR, and handle [pc]dfPrecision later.

Reply:
Yes, done, cdfPresision_ is already read-only so I could removed the mutable keyword.